stories/documentation/navigation/vertical-navigation/vertical-navigation-basic.stories.ts

```diff
@@ -33,19 +59,18 @@ function getTemplate(args: VerticalNavigationBasicStory): string {
 			</ul>
 		</li>
 		<li class="verticalNavigation-list-item">
-			<button class="verticalNavigation-list-item-link" aria-expanded="true">Item <span aria-hidden="true" class="lucca-icon icon-southArrow"></span></button>
+			<button class="verticalNavigation-list-item-link" aria-expanded="true"><span aria-hidden="true" class="verticalNavigation-list-item-link-icon lucca-icon icon-heart"></span>Item<span aria-hidden="true" class="lucca-icon icon-southArrow verticalNavigation-list-item-link-arrow"></span></button>
 			<ul class="verticalNavigation-list mod-child">
 				<li class="verticalNavigation-list-item">
 					<a href="#" class="verticalNavigation-list-item-link ${disabledLink}" ${disabledLinkTabIndex}>Item</a>
 				</li>
 				<li class="verticalNavigation-list-item">
-					<a href="#" class="verticalNavigation-list-item-link" aria-current="page">Item</a>
+					<a href="#" class="verticalNavigation-list-item-link">Item</a>
 				</li>
 			</ul>
 		</li>
 	</ul>
 </nav>`;
```

Comment:
It looks like the example only handles nested links. The links could also be on the parent list I think.